routes/authentication.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const router = express.Router();
+const { body, validationResult } = require('express-validator');
+
+const AuthController = require('../controllers/authentication.js');
+
+const authController = new AuthController();
```

Comment:
Esto genera una reserva de memoria que se ejecuta con el servidor aún cuando no se ha ejecutado, revisar.

Reply:
Lo pongo de esta manera
`
const AuthController = require('../controllers/authentication.js');

router.post(
  ...
      const response = await AuthController.loginUser(email, pass);
  ...
);
`

Pero me sale el error: "TypeError: AuthController.loginUser is not a function"